.nuget/create_nuget.py

```diff
@@ -36,9 +40,44 @@
         <dependencies> </dependencies>
     </metadata>
         <files>
+        <file src="build\**" target="build"/>
         <file src="runtimes\**" target="runtimes"/>
         </files>
     </package>
     ''' % version
+    prop_str = '''
+    <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <ItemGroup Condition="Exists('packages.config') OR
+                            Exists('$(MSBuildProjectName).packages.config') OR
+                            Exists('packages.$(MSBuildProjectName).config')">
+        <Content Include="$(MSBuildThisFileDirectory)\..\runtimes\win-x64\native\*.dll"
+                Condition="'$(PlatformTarget)' == 'x64'">
+        <Content Include="$(MSBuildThisFileDirectory)\..\runtimes\win-x64\native\*.exe"
```

Comment:
This is not valid MSBuild syntax (or even XML for that matter).

```xml
        <Content Include="$(MSBuildThisFileDirectory)\runtimes\win-x64\native\*.dll"
                Condition="'$(PlatformTarget)' == 'x64'">
        <Content Include="$(MSBuildThisFileDirectory)\runtimes\win-x64\native\*.exe"
                Condition="'$(PlatformTarget)' == 'x64'">
            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
            <Visible>false</Visible>
        </Content>
```

You have 2 open tags for `Content`, but only 1 closing tag.  Also `Content` items can't be nested.